bin/oneoff/data_fix/level_group_data

```diff
@@ -134,7 +173,8 @@ def main
     user_level, level_source_data = get_user_level_and_level_source_data(
       user_level_id
     )
-    next unless user_level && level_source_data
```

Comment:
As `get_user_level_and_level_source_data` was changed to throw an exception (in some prior version, it did not).